explore/src/main/scala/explore/highcharts.scala

```diff
@@ -6,4 +6,11 @@ package explore.highcharts
 import gpp.highcharts.mod._
 import japgolly.scalajs.react._
 
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+
 extension (c: Chart_) inline def showLoadingCB = CallbackTo(c.showLoading())
+
+@js.native
+@JSImport("highcharts/es-modules/masters/modules/accessibility.src.js", JSImport.Default)
+object HighchartsAccesibility extends js.Object
```

Comment:
Can we put this in the facade in lucuma-react?

Reply:
Yes we could move these there